src/components/Badges/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react"
+
+import * as S from "./styles"
+import { Props } from "./types"
+
+import { Box } from "../Box"
+
+export const Badges = ({
+    label,
+    type = "badge",
+    startIcon,
+    endIcon
+}: Props) => {
+    const [showCloseIcon, setShowCloseIcon] = useState(false)
+    const [closeTag, setCloseTag] = useState(false)
+
+    return (
+        <Box>
+            {type === "tag" ? (
+                <S.Tags
+                    onMouseEnter={() => setShowCloseIcon(!showCloseIcon)}
+                    onMouseLeave={() => setShowCloseIcon(!showCloseIcon)}
+                    style={closeTag ? { display: "none" } : {}}
+                    data-testid='testid-tag'
```

Comment:
1. Adicionar atributo `id` para compor test-id

```suggestion
                    data-testid={`tag-${id}`}
```